### PR TITLE
Opam publish ppx protocol conv.5.0.0

### PR DESCRIPTION
**Pull request by andersfugmann**

### CODE REVIEW

**Comment by andersfugmann** [comment]
The only failing test is the osx build which, as usual,  times out.

File: packages/ppx_protocol_conv_yaml/ppx_protocol_conv_yaml.5.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "ppx_protocol_conv" {= "5.0.0"}
+  "yaml"
+  "yaml" {with-test & >= "2.0.0"}
```
Comment (kit-ty-kate) [inline packages/ppx_protocol_conv_yaml/ppx_protocol_conv_yaml.5.0.0/opam]: yaml is present twice. Which one is true?

Comment (andersfugmann) [reply packages/ppx_protocol_conv_yaml/ppx_protocol_conv_yaml.5.0.0/opam]: Nice catch, Thanks.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**